src/com/ikhokha/techcheck/CommentAnalyzer.java

```diff
@@ -29,11 +29,13 @@ public Map<String, Integer> analyze() {
 					
 					incOccurrence(resultsMap, "SHORTER_THAN_15");
 
-				} else if (line.contains("Mover")) {
+				}
+				if (line.contains("Mover")) {
 
```

Comment:
removing the else if chaining makes sure that all the conditions are tested on each `loop` iteration